hc/settings.py

```diff
@@ -53,6 +53,7 @@
 
 AUTHENTICATION_BACKENDS = (
     'hc.accounts.backends.EmailBackend',
+    ('django.contrib.auth.backends.ModelBackend'),
```

Comment:
I think you should remove this line since one can already log in using their email, i would suggest you remove it since it might inconvenience us in the future.